upgrades/0.0.99.seperate-ns-from-app/makefile

```diff
@@ -40,3 +40,12 @@ lint: $(GOLANGCILINT)
 	$(GOLANGCILINT) run
 
 check: fmt lint test
+
+clean-upgrade:
+	find manual-tests -name 'upgrade' | xargs rm
+distribute-upgrade:
```

Comment:
This won't work without doing some manual steps (copying the IAC folders), perhaps add a comment about that?

Reply:
Yeah. I didn't want to make it perfect since we're probably not going to need it again, then again I didn't want to throw it away. A comment seems like a nice compromise